src/common/utils/Comparator.ts

```diff
@@ -0,0 +1,25 @@
+import { Data } from '../../common/types/TableTypes.types';
```

Comment:
If inside the file there is no class, then you should name it with camelCase notation

Reply:
Fixed in [19e41a7](https://github.com/dorm-social-room-manager/dsrm-frontend/pull/38/commits/19e41a7bee8fcb76076439e0e09b6f0f1996870b)